var/spack/repos/builtin/packages/basalt/package.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+# ----------------------------------------------------------------------------
+# If you submit this package back to Spack as a pull request,
+# please first remove this boilerplate and all FIXME comments.
+#
+# This is a template package file for Spack.  We've put "FIXME"
+# next to all the things you'll want to change. Once you've handled
+# them, you can save this file and test your package like this:
+#
+#     spack install basalt
+#
+# You can edit this file again by typing:
+#
+#     spack edit basalt
+#
+# See the Spack documentation for more information on packaging.
+# ----------------------------------------------------------------------------
+
+from spack import *
+
+
+class Basalt(PythonPackage):
+    """C++11 Graph Storage library"""
+
+    homepage = "https://github.com/tristan0x/basalt"
+    url      = "git@github.com:tristan0x/basalt.git"
+    
+    version('develop', git=url, branch='master', submodules=True, preferred=True)
+    version('0.1.1', git=url, tag='v0.1.1', submodules=True)
+
+    depends_on('py-setuptools', type=('build', 'run'))
```

Comment:
There are missing tests dependencies.

Reply:
Typically not a good idea to have develop version as preferred

Reply:
test dependencies should be added with `when='+test'`, IIRC?

Reply:
You can test this by building with `spack install --test=root`.

Reply:
I created a new PR for just the humanize package. I also encapsulated the change suggested by @ferdonline . 
@tristan0x I did not find today the missing test dependencies. I will check again tomorrow. 
@matz-e Thanks for pointing that out.
And thank all of you for your comments.

Reply:
Checked the tests that run correctly with the help of @matz-e :+1: